playroom/snippets.tsx

```diff
@@ -681,6 +681,60 @@ const cardSnippets: Array<Snippet> = [
             subtitle="Subtitle"
         />`,
     },
+    {
+        group: 'Cards',
+        name: 'DisplayDataCard',
+        code: `
+        <DisplayDataCard
+          icon={
+            <Circle size={40} backgroundColor={skinVars.colors.brandLow}>
```

Comment:
does skinVars works in playroom?

Reply:
hmmm... let me see

Reply:
yes, it works, but we can use just `colors`, so I'll change it